flink-end-to-end-tests/flink-glue-schema-registry-json-test/src/main/java/org/apache/flink/glue/schema/registry/test/json/GSRKinesisPubsubClient.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
Seems to be duplicated with the class in the `avro` module. Is there a way we can reuse rather than copy and paste?